CMakeLists.txt

```diff
@@ -884,9 +884,11 @@ if(NOT WIN32 OR ROCKSDB_INSTALL_ON_WINDOWS)
   )
 endif()
 
+add_subdirectory(third-party/gtest-1.8.1/fused-src/gtest)
```

Comment:
Should we add subdirectory only when we want to build with tests?

Reply:
Emm, that's fine. At first I put it beforehand just for convenience, since the `gtest` target is depended on `test` and `benchmarks`. I will make fix it to the original.